apps/backoffice-v2/src/domains/alerts/fetchers.ts

```diff
@@ -46,6 +46,52 @@ export const AlertTypes = [
   AlertType.UNUSUAL_PATTERN,
 ] as const satisfies ReadonlyArray<TObjectValues<typeof AlertType>>;
 
+export const AlertState = {
```

Comment:
Why dont we add an api that fetch those options in order to make our product aligned with backend options?
A query we can use to fetch options for an enum:
```sql
SELECT unnest(enum_range(NULL::"TransactionRecordType"))::text;
```

<img width="562" alt="image" src="https://github.com/ballerine-io/ballerine/assets/6435752/68266f78-d8f1-45a8-8753-ea2487a754c3">

Reply:
Because it would just give you the runtime values and no type information.

Reply:
what is type information?

Reply:
TypeScript type. It would regress to `string` at best. And even then there won't be an enum to access for logic.